components/hab/src/cli.rs

```diff
@@ -25,11 +25,11 @@ use protocol;
 use url::Url;
 
 use command::studio;
+use default_values::{
+    GOSSIP_DEFAULT_ADDR, GOSSIP_LISTEN_ADDRESS_ENVVAR, RING_ENVVAR, RING_KEY_ENVVAR,
+};
```

Comment:
I think these values can just live in `cli.rs` and be declared `pub`. Eventually I don't think any other code should need to reference them directly at which point `pub` can be removed.